test/framework/src/main/java/org/opensearch/snapshots/AbstractSnapshotIntegTestCase.java

```diff
@@ -352,7 +353,11 @@ protected void createRepository(String repoName, String type) {
 
     protected Settings.Builder randomRepositorySettings() {
         final Settings.Builder settings = Settings.builder();
-        settings.put("location", randomRepoPath()).put("compress", randomBoolean());
+        final boolean compress = randomBoolean();
+        settings.put("location", randomRepoPath()).put("compress", compress);
+        if (compress) {
+            settings.put("compression_type", randomFrom(CompressorType.values()));
```

Comment:
Same here: LZ4 testing is removed and no compression testing isn't covered.

Reply:
Same here I'm planning to add LZ4 as a separate PR if you do not mind.